src/orchestration/Orchestration.ts

```diff
@@ -228,7 +228,8 @@ export class Orchestration {
         {
             cookieAttributes: configCookieAttributes,
             ...partialConfig
-        }: PartialConfig = {}
+        }: PartialConfig = {},
+        isFromScript?: boolean | undefined
```

Comment:
suggestion: Instead of adding a new parameter, use the `config` object.

Options to consider:
1. Add `"aws:client": "arw-module"` to the `sessionAttributes` object.
2. Add a new field called `client` to the config object. Set the default value of `client` to `arw-module`.

(1) would require having the application-provided `aws:` attributes overwrite the default `aws:` attributes, which I still think we should do. The advantage of (1) is that we don't need to add any new fields to the `config` object.

Reply:
suggestion (alternate): Instead of adding a new parameter, use the build script to differentiate between the two at build time.

For example, during build time we could (a) set an environment variable containing the client name and version, or (b) generate a file containing the client name and version.

I'm hesitant to add a new parameter to the constructor.

Reply:
Added new field `client` to config object with default value set to `arw-module`